website/src/components/DataTable/DataTable.tsx

```diff
@@ -50,6 +50,8 @@ export type FilterItem = {
   value: string;
 };
 
+export type FilterOption = "display_name" | "username";
```

Comment:
I'm not sure why we need this, but it seem incorrect. The DataTable component is generic, it should not know column filter name

Reply:
Ya sorry, I have removed that type